### DISCUSSION TITLE
NetBox Chart

### DISCUSSION THREAD

**Discussion by bootc** [netbox-community/netbox-docker#1160] [General]
I "maintain" the NetBox Helm chart at https://github.com/bootc/netbox-chart. I clearly don't have the time or inclination to actually maintain it any more, but I think it would be a shame if it just bit-rotted to death. Is there any interest in it becoming a NetBox Community project so that it can live on?

**Comment by Kerwood** [comment]
As a user of your Netbox Chart, I appreciate you taking the time to try and tranfer the project to some one else instead of just archiving your repo, or even worse, just letting it sit unresponsive. 

I sure hope that the Netbox Community have the time and resources to take your Netbox Chart in and maintain it for future use.

**Comment by K4pper** [comment]
Really nice that you are trying to transfer the project.
I also hope the Netbox Community have the resources to maintain the chart

**Comment by moerafiee** [comment]
recently started to use this chart, this is a great project, do we have any timeline or plan to transfer tis project?

**Comment by arichard42** [comment]
It seams this fork is active :
https://github.com/startechnica/apps/tree/main/charts/netbox

**Comment by bootc** [comment]
Just a quick note that someone from `netbox-community` has reached out to me and plans are being hatched to transfer the repository under https://github.com/netbox-community. The intention is that the chart would be actively maintained, will remain open source (of course), and I'll also remain a maintainer of the project in its new home.

**Comment by RangerRick** [comment]
Just to cover all the bases (we've had this conversation in 3 different places 😅), but I've been brought on by NBL to help maintain packaging and release stuff, including the helm chart. We've finished the process of moving it over to `netbox-community`, and I have been going through existing commits and issues to figure out what needs to be done to get things straightened out.

@bootc has been a huge help getting us here, and @LeoColomb is in the middle of contributing a bunch of great quality-of-life improvements to the workflow so we can be in a good place to maintain it going forward.

Thanks to everyone for your patience!
